README.md

```diff
@@ -42,7 +42,11 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
+S -> A S A  
+S -> A  
+A -> 'r'  
+A -> 'g'  
+A -> 'b'
```

Comment:
# 0

The problem is that in the first rule, the A's can expand to different colors, so there's no symmetry.

You'd need something like

       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b